jujugui/static/gui/src/app/components/profile/model-list/model-list.js

```diff
@@ -156,135 +167,122 @@ class ProfileModelList extends React.Component {
     @return {Object} The model list as JSX.
   */
   _generateModels() {
-    const icons = new Map([
-      ['read', 'show_16'],
-      ['write', 'edit_16'],
-      ['admin', 'user_16']
-    ]);
+    const icons = new Map([['read', 'show_16'], ['write', 'edit_16'], ['admin', 'user_16']]);
```

Comment:
no

Reply:
Why doesn't your Eslint rules catch this? I've everything set up to fix on save? I guess I still need to guess how you like certain code formatted 🤦🏻‍♂️